### DISCUSSION TITLE
GoodTask

### DISCUSSION THREAD

**Discussion by Tnixc** [catppuccin/catppuccin#1842] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
Add any other context or screenshots about the port request here.

![image](https://user-images.githubusercontent.com/85466117/193438845-eb3039ff-7c27-4fd9-9440-657015fc7118.jpeg)

**Comment by nekowinston** [comment]
Looking good!

Please leave a new comment when the repo is up 🙂

**Comment by Tnixc** [comment]
I don't use this anymore. If anyone would like to pick this up they can do so.
